* Set THREAD_SIZE to 16K, so that ndiswrapper can handle larger Windoze
drivers. Smaller 8K stacks cannot handle some drivers being loaded.
diff -urN linux-2.6.21/include/asm-i386/thread_info.h linux-2.6.21-16kstacks/include/asm-i386/thread_info.h
--- linux-2.6.21/include/asm-x86/thread_info_32.h 2007-04-25 23:08:32.000000000 -0400
+++ linux-2.6.21-16kstacks/include/asm-x86/thread_info_32.h   2007-04-26 14:15:55.000000000 -0400
@@ -54,9 +54,5 @@
 
 #define PREEMPT_ACTIVE		0x10000000
-#ifdef CONFIG_4KSTACKS
-#define THREAD_SIZE            (4096)
-#else
-#define THREAD_SIZE		(8192)
-#endif
+#define THREAD_SIZE        (16384)
 
 #define STACK_WARN             (THREAD_SIZE/8)
